templates/connect.php

```diff
@@ -525,6 +552,11 @@ class="button button-secondary" tabindex="2"><?php fs_esc_html_echo_x_inline( 'S
 				$( document.body ).addClass( 'fs-loading' );
 			};
 
+            $('#fs_personal_data input').on('focus', function() {

```

Comment:
We also need spaces here (similar to PHP). There are examples here:
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#examples-of-good-spacing

So it should be like this:
```
$( '#fs_personal_data input' ).on( 'focus', function() {
    $( '#fs_personal_data' ).removeClass( 'error' );
    resetLoadingMode();
} );
```